pkg/config/testdata/TestLoadConfigSuccess/config.golden.yaml

```diff
@@ -8,17 +8,15 @@ sources:
                 pod:
                     noLatestImageTag: false
                     labelsSet: true
+            events:
```

Comment:
If now we have global `namespace.include`, should it be also visible in the golden file? I can't see it 🤔

Reply:
There was a typo (missing `s`), should be `namespaces` instead. Fixed 👍